cmake/emu/qemu.cmake

```diff
@@ -328,7 +328,7 @@ foreach(target ${qemu_targets})
     ${QEMU_SMP_FLAGS}
     ${QEMU_KERNEL_OPTION}
     DEPENDS ${logical_target_for_zephyr_elf}
-    WORKING_DIRECTORY ${APPLICATION_BINARY_DIR}
+    WORKING_DIRECTORY ${APPLICATION_BINARY_DIR}/zephyr
```

Comment:
As we provide full path to all elf-files, then i don't believe it matters if `qemu` itself is launched with working directory as `APPLICATION_BINARY_DIR`, or with `ZEPHYR_BINARY_DIR`.

But users may supply extra flags using environment setting `QEMU_EXTRA_FLAGS`, and if they do `-D<logfile>`, then it might be easier to find such log file directly in `APPLICATION_BINARY_DIR` than under `ZEPHYR_BINARY_DIR`